### ISSUE TITLE
[fvt] 2.13.5 rvitals need to modify man page

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#3126]
env:all os
build:root@fs1 ~]# lsxcatd -v
Version 2.13.4 (git commit 305ec7d71e4333f43f325e546daaf97fa943cd56, built Wed May 17 06:15:47 EDT 2017)
[root@fs1 ~]# 
How to reproduce:
```
1.man rvitals
 BMC specific:
       rvitals noderange {temp|voltage|wattage|fanspeed|power|leds|all}
2.But when I use:

[root@fs1 ~]# rvitals  frame45cn07 cputemp
frame45cn07: Ambient Temp: 27 C (81 F)
frame45cn07: CPU Core Temp 1: 53 C (127 F)
frame45cn07: CPU Core Temp 10: 55 C (131 F)
frame45cn07: CPU Core Temp 11: 54 C (129 F)
frame45cn07: CPU Core Temp 12: 0 C (32 F)
frame45cn07: CPU Core Temp 13: 52 C (126 F)
frame45cn07: CPU Core Temp 14: 0 C (32 F)
frame45cn07: CPU Core Temp 15: 52 C (126 F)
frame45cn07: CPU Core Temp 16: 51 C (124 F)
frame45cn07: CPU Core Temp 17: 53 C (127 F)
frame45cn07: CPU Core Temp 18: 52 C (126 F)
frame45cn07: CPU Core Temp 19: 0 C (32 F)
frame45cn07: CPU Core Temp 2: 53 C (127 F)
frame45cn07: CPU Core Temp 20: 51 C (124 F)
frame45cn07: CPU Core Temp 21: 52 C (126 F)
frame45cn07: CPU Core Temp 22: 53 C (127 F)
frame45cn07: CPU Core Temp 23: 52 C (126 F)
frame45cn07: CPU Core Temp 24: 53 C (127 F)
frame45cn07: CPU Core Temp 3: 0 C (32 F)
frame45cn07: CPU Core Temp 4: 54 C (129 F)
frame45cn07: CPU Core Temp 5: 55 C (131 F)
frame45cn07: CPU Core Temp 6: 55 C (131 F)
frame45cn07: CPU Core Temp 7: 55 C (131 F)
frame45cn07: CPU Core Temp 8: 53 C (127 F)
frame45cn07: CPU Core Temp 9: 55 C (131 F)
frame45cn07: CPU Diode 1: 36 C (97 F)
frame45cn07: CPU Diode 2: 36 C (97 F)
frame45cn07: CPU1 Temp: 55 C (131 F)
frame45cn07: CPU2 Temp: 53 C (127 F)
frame45cn07: DIMM1 Temp: 33 C (91 F)
frame45cn07: DIMM10 Temp: 32 C (90 F)
frame45cn07: DIMM11 Temp: 32 C (90 F)
frame45cn07: DIMM12 Temp: 32 C (90 F)
frame45cn07: DIMM13 Temp: 31 C (88 F)
frame45cn07: DIMM14 Temp: 31 C (88 F)
frame45cn07: DIMM15 Temp: 32 C (90 F)
3.Node definiton:[root@fs1 ~]# lsdef frame45cn07
Object name: frame45cn07
    arch=ppc64le
    bmc=50.45.7.1
    bmcvlantag=551
    chain=runcmd=date,osimage=rhels7.3-GA-ppc64le-netboot-compute
    cons=ipmi
    cpucount=160
    cputype=POWER8 (raw), altivec supported
    currchain=osimage=rhels7.3-GA-ppc64le-netboot-compute
    currstate=osimage=rhels7.3-GA-ppc64le-netboot-compute
    disksize=sda:1000GB,sdb:1000GB
    groups=all,disco,compute,frame45,firestone
    installnic=mac
    ip=192.168.45.7
    mac=98:be:94:59:ea:27
    memory=261482MB
    mgt=ipmi
    mtm=8335-GTA
    netboot=petitboot
    nodetype=mp
    os=rhels7.3-GA
    postbootscripts=otherpkgs
    postscripts=syslog,remoteshell,syncfiles
    profile=compute
    provmethod=rhels7.3-GA-ppc64le-netboot-compute
    serial=1210532
    status=success updating firmware
    statustime=04-12-2017 06:14:31
    supportedarchs=ppc64
    switch=frame45sw1
    switchport=7


```

**Comment by gurevichmark** [comment]
@junxiawang I do not think this is a problem with man page.

The problem is that `rvitals` command does not verify if specified option is a valid one.
The command appears to accept any string, as long as it contains a valid option in it.
For example, the following options generate identical output: `temp`, `testtemp`, `temptest`, `testtemptest`.
Same is true for all other `rvitals` options: `fan`, `power`, etc. (`junkfantestme`, `thisismypower`, etc. )

**Comment by whowutwut** [comment]
@gurevichmark We could fix this with similar processing that we have in `openbmc.pm` 

```
 458     } elsif ($command eq "rvitals") {
 459         $check = unsupported($callback); if (ref($check) eq "ARRAY") { return $check; }
 460         $subcommand = "all" if (!defined($ARGV[0]));
 461         unless ($subcommand =~ /^temp$|^voltage$|^wattage$|^fanspeed$|^power$|^leds$|^all$/) {
 462             return ([ 1, "Unsupported command: $command $subcommand" ]);
 463         }
```

**Comment by zet809** [comment]
Hi, @gurevichmark , will you pls help to fix this issue if you have time? It is low priority.

**Comment by gurevichmark** [comment]
Fixed by PR #3199.
Exposed option `chassis` which was handled by code, but was not shown in usage or man page.

**Comment by junxiawang** [comment]
verified
